### ISSUE TITLE
Path to MKDOCS_EXECUTABLE contains backslashes

### ISSUE THREAD

**Issue by dzenanz** [BlueQuartzSoftware/DREAM3DSuperbuild#11]
The generated file `DREAM3D_SDK.cmake` contains a line which sets `MKDOCS_EXECUTABLE` to full quoted path. This path contains backslashes, whereas CMake expects forward slashes. To go to the next step, I had to manually change `\` into `/`, e.g.:
```text
set(MKDOCS_EXECUTABLE "C:/Users/Dzenan/AppData/Roaming/Python/Python37/site-packages/mkdocs/__init__.py")
```

**Comment by dzenanz** [comment]
To backtrack a little, I had to `pip install` mkdocs package, and since I was doing it from within ordinary console window I had to use -user option (hence it ended up in `AppData`).

**Comment by imikejackson** [comment]
Fixed in commit 89ef20f421e50402de82e8fea1aa9df2e76c56c1

We now start with the Python executable and derive the path to MkDocs inside of the Scripts folder. this at least works on Anaconda installations. When others get errors we can tackle their installations.
